scenarios/README.md

```diff
@@ -0,0 +1,82 @@
+# Scenarios
+
+This folder contains various projects conducted in cyclus.
+
+## 2014-E&S
+Summary: Transition scenarios for evaluation groups from
+the 2014 E&S Study from the U.S. Department of Energy.
+
+Wigeland, R., T. Taiwo, H. Ludewig, M. Todosow, W. Halsey, J. Gehin, R. Jubin, J. Buelt,
+S. Stockinger, K. Jenni, B. Oakley (2014), “Nuclear Fuel Cycle Evaluation and Screening – Final
+Report,” FCRD-FCO-2014-000106, Fuel Cycle Technologies Programme.
+
```

Comment:
```suggestion
Summary: Transition scenarios for evaluation groups from
the 2014 E&S Study from the U.S. Department of Energy:
Wigeland, R., T. Taiwo, H. Ludewig, M. Todosow, W. Halsey, J. Gehin, R. Jubin, J. Buelt,
S. Stockinger, K. Jenni, B. Oakley (2014), “Nuclear Fuel Cycle Evaluation and Screening – Final
Report,” FCRD-FCO-2014-000106, Fuel Cycle Technologies Programme.

```

Reply:
To make it more clear that that's the E&S Study. If you feel like it was clear the initial way feel free to ignore this.